internal/statestore/backfill.go

```diff
@@ -243,19 +352,38 @@ func (rb *redisBackend) GetIndexedBackfills(ctx context.Context) (map[string]int
 	}
 	defer handleConnectionClose(&redisConn)
 
-	bfIndex, err := redis.StringMap(redisConn.Do("HGETALL", allBackfills))
+	ttl := getBackfillReleaseTimeout(rb.cfg)
+	curTime := time.Now()
+	endTimeInt := curTime.Add(time.Hour).UnixNano()
+	startTimeInt := curTime.Add(-ttl).UnixNano()
+
+	acknowledgedIds, err := redis.Strings(redisConn.Do("ZRANGEBYSCORE", backfillLastAckTime, startTimeInt, endTimeInt))
```

Comment:
Since we are adding this "query" with zrangebyscore, shouldn't be a ZADD in indexing?? And the corresponding command when Deindexing backfills??

Reply:
We are actually already performing `ZADD` in StateStore AcknowledgeBackfill() function

Reply:
I think I had an error while merging all PRs together.

Reply:
Or this link
https://github.com/aLekSer/open-match/blob/26d1aa236a5238b1387e91d506d21ed09f3891cc/internal/statestore/backfill.go#L160